src/pretix/control/forms/orders.py

```diff
@@ -13,7 +13,10 @@ class Meta:
         model = Order
         fields = ['expires']
         widgets = {
-            'expires': forms.DateInput(attrs={'class': 'datepickerfield'}),
+            'expires': forms.DateInput(attrs={
+                'class': 'datepickerfield',
+                'data-payment-weekdays-disabled': '[]'
```

Comment:
I agree, `data-is-payment-date=true` is much more clearer. Let me update my branch with this suggestion. Thank you!